src/main/java/com/plivo/api/models/address/Address.java

```diff
@@ -0,0 +1,181 @@
+package com.plivo.api.models.address;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.plivo.api.models.base.BaseResource;
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class Address extends BaseResource {
```

Comment:
In address GET, List response "account" and "document_details" missing"

`{"id":"22754037731962","country_iso":"US","salutation":"Mr","first_name":"abhineet_test","last_name":"test","address_line1":"201","address_line2":"MISSION ST","city":"SAN FRANCISCO","region":"CA","postal_code":"94105","validation_status":"accepted","verification_status":"rejected","url":"https://api.plivo.com/v1/Account/MANTQ4ZMIZNWRLYZMXOD/Verification/Download/Document/22754037731962/","address_proof_type":"address"}`

https://api-reference.plivo.com/latest/curl/resources/verification/address/list-all-addresses

Reply:
as discussed submitted a PR for the same